app/lib/theme/widgets/ReadOnlyNumberWidget.tsx

```diff
@@ -18,24 +19,32 @@ const ReadOnlyNumberWidget: React.FC<WidgetProps> = ({
   return (
     <dd>
       {value ? (
```

Comment:
This `value ? ` conditional is causing the summary (and any other readOnly view) to show 'Not Added' for values of 0

Reply:
@dleard Thanks for catching this 🙏 This led to a bigger problem with saving zero value and our diffing.